client/src/templates/Introduction/intro.css

```diff
@@ -269,6 +269,7 @@ button.map-title {
 .map-challenges-grid {
   display: flex;
   flex-wrap: wrap;
+  justify-content: space-evenly;
   margin-top: 1rem;
 }
```

Comment:
Adding a `column-gap` of 2-3 pixels seems to space out the steps evenly, and ensures the bottom row is still aligned with the rest of the grid. Some slight padding can also make the gaps look a bit more even:

```suggestion
.map-challenges-grid {
  display: flex;
  flex-wrap: wrap;
  padding: 5px;
  column-gap: 2.5px;
  margin-top: 1rem;
}
```

Which looks like this:

![image](https://user-images.githubusercontent.com/2051070/183351093-f1d032cf-c8b5-44d7-9557-a89024c3ad52.png)

Reply:
But there's still some extra space on the right due to the wrapping. This might be a good chance to use CSS Grid instead of Flexbox, which seems to handle this better. But using Grid would require some changes in a different rule:

```css
.map-challenge-title {
  list-style: none;
  font-size: 1.13rem;
}

...

.map-challenges-grid {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-evenly;
  margin-top: 1rem;
}
```

![image](https://user-images.githubusercontent.com/2051070/183351889-d674cadb-3da5-47ee-8c3e-65aeaa678bfb.png)

Reply:
This looks great.

Reply:
I literally called these "grid items" in the PR description and didn't think to use CSS Grid. 🤦🏻

Reply:
No worries! And looking at my comment above I originally left another Flexbox suggestion, not one with CSS Grid that I tested locally haha 🤦🏻

@bbsmooth's suggestion looks great, and is less complex than the one I tested.